lib/app-router.js

```diff
@@ -117,6 +154,7 @@ module.exports = function (router) {
               message: `Error: ${commandErr}.`
             });
           } else {
+            console.log(commandResult);
```

Comment:
Remove

Reply:
agreed, done.  aside: Any thoughts to what you'd want the output from apex scripts to look like?  Right now, the UI just informs that each one ran.